Online/State/RealizedPlayerState.cs

```diff
@@ -58,6 +68,16 @@ public RealizedPlayerState(OnlineCreature onlineEntity) : base(onlineEntity)
                 tongueRequestedLength = tongue.requestedRopeLength;
                 tongueAttachedChunk = BodyChunkRef.FromBodyChunk(tongue.attachedChunk);
             }
+
+            if ((p.graphicsModule as PlayerGraphics).tail != null)
```

Comment:
I have a gist this might nullref when a player enters-from-offscreen and spawns in the pipes system.

Having the whole thing under `if (p.graphicsModule is PlayerGraphics playerGraphics)` lets you be safe and reuse the cast.